.github/renovate.json5

```
@@ -1,10 +1,18 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "dependencyDashboard": true,
-  "suppressNotifications": ["prEditedNotification"],
-  "extends": ["config:recommended"],
-  "labels": ["internal"],
-  "schedule": ["before 4am on Monday"],
+  "suppressNotifications": [
+    "prEditedNotification"
+  ],
+  "extends": [
+    "config:recommended"
+  ],
+  "labels": [
+    "internal"
+  ],
+  "schedule": [
+    "before 4am on Monday"
```

nit: this is a `.json5` file, not `.json`, so we can use trailing commas to reduce diffs from future changes to this file:

```suggestion
  "suppressNotifications": [
    "prEditedNotification",
  ],
  "extends": [
    "config:recommended",
  ],
  "labels": [
    "internal",
  ],
  "schedule": [
    "before 4am on Monday",
```

Did prettier or another autoformatter make the changes to these lines?

---

It was jetbrains. i disabled the autoformatter for JSON and instead ran prettier (which is what we use for other files)

---

I like the prettier changes more than the JetBrains changes :) though it's a _little_ hard to see here which changes have an impact on the config and which are just cosmetic